0.4/examples/invalid/invalid_axis_type.json

```diff
@@ -0,0 +1,33 @@
+{
+    "multiscales": [
+        {
+            "axes": [
+                {
+                    "name": "y",
+                    "type": "invalid",
```

Comment:
This is currently a SHOULD, see https://github.com/ome/ngff/blob/main/latest/index.bs#L217. Not sure how exactly we want to tackle this. I will leave a more general comment once I went through everything.

Reply:
Agreed,  this should not fail validation but probably be flagged by the stricter schema. I propose to open a follow-up PR with the infrastructure to differentiate the two of them.

Reply:
Actually, re-reading this example, this should remain invalid.

As you rightfully mentioned `type` is a SHOULD and that is not sufficient ground to fail the strict schema. However there is only one `space` axe within the `multiscales` dictionary

Reply:
yes, you're right. But the name of this example is a bit misleading then.

Reply:
And in fact it is failing because `type` is not `space, channel or time` rather than only-1-space-axis.
So the schema should allow 1 'invalid' axis.

Reply:
> And in fact it is failing because type is not space, channel or time rather than only-1-space-axis.
So the schema should allow 1 'invalid' axis.

I am not sure what you mean by this @will-moore. But I agree that this one should fail the strict spec because it has only 1 spatial axis, as @sbesson points out. But I would maybe rename the example to make that clear.

Reply:
Nods, that's probably more tests worth injecting but my understanding of the `type` restrictions is the following for `multiscales.axes`:

- 1 space axes + anything else: invalid
- 2 (or 3) space axes, 1 channel and/or 1 time: valid
- 2 (or 3) space axes, 1 channel or 1 time), 1 custom type: valid but failing the strict schema
- 2 (or 3) space axes, 1 channel or 1 time, 1 unspecified: valid but failing the strict schema
- 2 (or 3) space axes, 2 customs or 2 unspecified or (1 custom and 1 unspecified): valid but failing the strict schema
- 4+ space axes + anything else: invalid

Reply:
> 2 (or 3) space axes, 1 channel or 1 time, 1 unspecified: valid but failing the strict schema

with 3 spatial axes this would fail because we would have > 5 dim. Same for some of the other examples you bring up.
But otherwise I agree with your list.

Reply:
Heh the `or` in between 1 channel and 1 time was meant to capture that but I certainly failed to write the different combinations in a concise way. Expanding a bit, all the following scenarios are valid according to the minimal schema but should fail the strict schema:

- 2 space, 1 channel, 1 unspecified
- 2 space, 1 time, 1 unspecified
- 3 space, 1 channel, 1 unspecified
- 3 space, 1 time, 1 unspecified